src/NotesLoaderSM.cpp

```diff
@@ -128,7 +128,7 @@ void SMSetSampleLength(SMSongTagInfo& info)
 void SMSetDisplayBPM(SMSongTagInfo& info)
 {
 	// #DISPLAYBPM:[xxx][xxx:xxx]|[*];
-	if((*info.params)[1] == "*")
+	if((*info.params)[1] == "*" || (*info.params)[1] == "?")
```

Comment:
While this should resolve the issue with `#DISPLAYBPM:?`, this is not allowed by the spec: https://github.com/stepmania/stepmania/wiki/sm#displaybpm

Should we add it to the spec?

Reply:
I should note that the wiki page on the sm format should not currently be treated as spec.  

I authored much of that wiki page with the hope of guiding stepartists and general users, not to define the spec.

Reply:
Would you edit the wiki page then?
According to @shakesoda, this should be in there: https://github.com/stepmania/stepmania/pull/1888#issuecomment-532488594